anvil/build.gradle

```diff
@@ -30,16 +15,72 @@ android {
 			minSdkVersion 21
 		}
 	}
-
-	testOptions {
-		unitTests.returnDefaultValues = true
-	}
 }
 
-apply from: 'attr-quirks.gradle'
+def nop = { attr -> null }
+
 anvilgen {
 	type = 'sdk'
-	quirks = QUIRKS
+	quirks = [
```

Comment:
What is reasoning behind moving those quirks into the same file? IMHO it is cleaner when those nasty hacks are left in separate file

Reply:
I've felt that with reduced size of build.gradle files it's okay to migrate quirks to the main one. I'm also hoping to reduce quirks themselves by migrating most of the code into, well, code, leaving only switches in this map to rename and disable stuff.

If you think we should still leave them separate, let me know. I don't have strong opinion on this.

Reply:
Sounds good!